Shared/Samples/Show coordinates in multiple formats/ShowCoordinatesInMultipleFormatsView.swift

```diff
@@ -0,0 +1,167 @@
+            )
+            .onSubmit {
+                model.updateMapPoint(
+                    point: CoordinateFormatter.point(
+                        fromLatitudeLongitudeString: model.latLongDMSTextField,
+                        spatialReference: model.mapPoint.spatialReference
+                    )!)
+            }
+            CoordinateTextField(
+                title: "UTM",
+                text: $model.utmTextField
+            )
+            .onSubmit {
+                model.updateMapPoint(
+                    point: CoordinateFormatter.point(
+                        fromUTMString: model.utmTextField,
+                        spatialReference: model.mapPoint.spatialReference,
+                        conversionMode: .latitudeBandIndicators
+                    )!)
+            }
+            CoordinateTextField(
+                title: "USNG",
+                text: $model.usngTextField
+            )
+            .onSubmit {
+                model.updateMapPoint(
+                    point: CoordinateFormatter.point(
+                        fromUSNGString: model.usngTextField,
+                        spatialReference: model.mapPoint.spatialReference
+                    )!)
+            }
+            
+            // Create a map view to display the map.
+            MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+                .onSingleTapGesture { _, mapPoint in
+                    model.updateMapPoint(point: mapPoint)
+                }
+        }
+    }
+}
+
+struct CoordinateTextField: View {
+    /// The title string.
+    var title: String
+    
+    /// The text string.
+    @Binding var text: String
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .padding([.leading, .top], 8)
+                .padding(.bottom, -5)
+            TextField("", text: $text)
+                .textInputAutocapitalization(.never)
+                .disableAutocorrection(true)
+                .textFieldStyle(.roundedBorder)
+                .padding([.leading, .trailing, .bottom], 8)
+        }
+    }
+}
+
+private extension ShowCoordinatesInMultipleFormatsView {
+    // The view model for the sample.
+    class Model: ObservableObject {
+        /// A map with an imagery basemap.
+        var map = Map(basemapStyle: .arcGISImageryStandard)
+        
+        /// The graphics overlay for the point graphic.
+        lazy var graphicsOverlay = GraphicsOverlay(graphics: [pointGraphic])
+        
+        /// A yellow cross graphic for the map point.
+        private var pointGraphic: Graphic = {
+            let yellowCrossSymbol = SimpleMarkerSymbol(style: .cross, color: .yellow, size: 20)
+            return Graphic(symbol: yellowCrossSymbol)
+        }()
+        
+        /// The point on the map.
+        @Published var mapPoint: Point!
+        
+        /// The decimal degrees text.
+        @Published var latLongDDTextField = ""
```

Comment:
Instead of "…TextField", change their names to "…String" to make them read better